### DISCUSSION TITLE
Deployment error

### DISCUSSION THREAD

**Discussion by 2-GO** [community/community#56346] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello I have tried to deploy a web-portfolio on gh pages and encountered and error. I have tried multiple solutions but can't seem to get it to work. Does anyone have any suggestions or fixes? I would really appreciate it. 

Github repository link to my [code](https://github.com/2-GO/web-portfolio)

![image](https://github.com/community/community/assets/102247107/25cdeed9-1305-434d-830f-889fc74e1ae8)

**Comment by airtower-luna** [comment]
Your `index.html` uses absolute links starting with `/2-GO/web-portfolio/`, when your site is deployed at `/web-portfolio/`. The best solution would be to use relative paths, but fixing the absolute ones should work, too.

See also: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#absolute_urls_vs._relative_urls

**Reply by 2-GO** [reply]
Hey [airtower-luna], when I inspect the website I can see the links starting with "/2-GO/web-portfolio/".  Also,  I read the post you attached and understand the difference between absolute and relative links ,but I cant seem to find where to apply this fix within my document files. Can you please elaborate a little more? I would really appreciate it.

**Reply by airtower-luna** [reply, marked as answer]
Your page gets deployed to `https://2-go.github.io/web-portfolio/`, because the repository is named `web-portfolio`. So for example the `favicon.ico` will end up at `https://2-go.github.io/web-portfolio/favicon.ico`.

But your [`index.html`](https://github.com/2-GO/web-portfolio/blob/3d808e896b9a6d67efd4211e2e30c39539dd789b/index.html) has (hard to link directly, because it lacks proper line breaks :slightly_frowning_face:):

```html
<link rel="icon" href="/2-GO/web-portfolio/favicon.ico"/>
```

`/2-GO/` and everything underneath simply doesn't exist on the server. Best would be to use a relative link (in the example: `href="favicon.ico"`), or alternatively fix the absolute path.

**Reply by 2-GO** [reply]
Thank you for your time @airtower-luna I got the website to finally show up, not as intended but I just took a step in the right direction. You helped me publish my first web-portfolio (although unfinished atm) I really appreciate you.
